pycaret/containers/models/time_series.py

```diff
@@ -376,31 +410,23 @@ def __init__(self, globals_dict: dict) -> None:
             tune_distribution=tune_distributions,
             tune_args=tune_args,
             is_gpu_enabled=self.gpu_imported,
+            eq_function=eq_function,
         )
 
     @property
     def _set_args(self) -> Dict[str, Any]:
-        args = {"sp": self.sp} if self.seasonality_present else {}
+        args = {"sp": self.sp}
         return args
 
     @property
     def _set_tune_grid(self) -> Dict[str, List[Any]]:
-        if self.seasonality_present:
-            tune_grid = {
-                "strategy": ["last", "mean", "drift"],
-                "sp": [self.sp, 2 * self.sp],
-                # Removing fh for now since it can be less than sp which causes an error
-                # Will need to add checks for it later if we want to incorporate it
-                "window_length": [None],  # , len(fh)]
-            }
-        else:
-            tune_grid = {
-                "strategy": ["last", "mean", "drift"],
-                "sp": [1],
-                # Removing fh for now since it can be less than sp which causes an error
-                # Will need to add checks for it later if we want to incorporate it
-                "window_length": [None],  # , len(fh)]
-            }
+        tune_grid = {
+            "strategy": ["last", "mean"],
+            "sp": [self.sp, 2 * self.sp],
```

Comment:
Is it possible for sp to be 1 here?

Reply:
Do I need to disable this model container when seasonality is 1?

Reply:
I think that would be best